test/Datadog.Trace.Tests/DogStatsDTests.cs

```diff
@@ -27,7 +28,7 @@ public void Do_not_send_metrics_when_disabled()
             var statsd = new Mock<IDogStatsd>();
             var spans = SendSpan(tracerMetricsEnabled: false, statsd.Object);
 
-            Assert.True(spans.Count == 1, "Expected one span");
+            Assert.True(spans.Count == 1, AssertionFailureMessage(1, spans));
```

Comment:
Y'all can blame me for not using `Assert.Single(..)` in the first place (git blame says I wrote this 14 months ago 😅)